markdown/en/ff-middleware.md

```diff
@@ -0,0 +1,127 @@
+## Middleware
+
+Starting from version `1.2.12` FACT-Finder Web Components support the concept of middleware. You can register and configure pre-defined modules to manipulate the data being exchanged between Web Components and FACT-Finder.
```

Comment:
I'd prefer `Starting with version ...` instead of `Starting from version ...`.

Reply:
Pretty difficult one. And quite difficult to find some nice information on this topic.
I liked _PaulQ's_ answer of [this thread](https://forum.wordreference.com/threads/start-from-vs-start-with.2546655/) quite much. In that case I would favour `Starting from` because to me a version number feels like a part of a sequence from with we are planning to move away.

This [StackExchange](https://english.stackexchange.com/questions/112972/starting-with-vs-starting-from) thread is maybe not so suitble for our context but might solidify the favour for one or the other nevertheless.

Reply:
I see your point, it makes sense. The other version just sounds more natural to *me*.